.github/CODEOWNERS

```diff
@@ -111,13 +111,13 @@
 /sdk/eventhubs/                                      @conniey @anuchandy @lmolkova
 
 # PRLabel: %Event Hubs
-/sdk/eventhubs/microsoft-azure-eventhubs-eph/        @JamesBirdsall @sjkwak
+/sdk/eventhubs/microsoft-azure-eventhubs-eph/        @sjkwak
```

Comment:
@conniey @anuchandy Seems James no longer in Micrsoft. Can I safely delete it? Somehow it is only code owner failed my PR pipeline even there are more invalid users.

Reply:
@mssfang I fixed this in main, just pull the updated CODEOWNERS into your branch and you'll be good to go.

Reply:
Thanks Jim